playground/server/index.js

```diff
@@ -106,7 +106,13 @@ io.on('connection', socket => {
     };
     try {
       pushLog('debug', '## start connect()');
-      const initState = await sourceImplementation.connect(requestLogin, requestWebView, logger);
+      const initState = await sourceImplementation.connect(
+        {
+          requestLogin,
```

Comment:
can you add 

`
{
  requestLogin,
  requestToken,
  requestWebView,
}
`

to make it clear that there is also the possibility of inputing a token now? 

Thanks :)

Reply:
Will look at adding the requestToken param tonight.

I've also managed to revert the lock file without getting the dependency issues I encountered the first time around.